dockerfile_to_osbs.sh

```diff
@@ -15,8 +15,7 @@ function backup_dockerfile() {
 function get_available_operators() {
     # Available operators are determined by the list of .env files in the
     # folder that consists of all vars
-    local basedir=$(dirname "$0")
-    available_operators=$(basename -s .env ${basedir}/${OSBS_VARS_DIR}/*.env | grep -v default)
+    available_operators=$(basename -s .env ${OSBS_VARS_DIR}/*.env | grep -v default)
```

Comment:
Issues is that `get_available_operators` is returning nothing since `${OSBS_VARS_DIR}` (set on line 3) already includes ${basedir}